css/homepage.css

```diff
@@ -0,0 +1,70 @@
+@import url("https://fonts.googleapis.com/css2?family=Dancing+Script&family=Gideon+Roman&family=Libre+Baskerville&family=Nunito:wght@200&display=swap");
+
+.display-block {
+  padding: var(--spacing-md);
+  gap: var(--spacing-xs);
+}
+
+/* different categories of items to browse by */
+.category-card {
+  width: 15%;
+  overflow: hidden;
+  font-family: "Gideon Roman", cursive;
+}
+
+/* inner of the card to implement the hover functionality */
+.inner-category-card:hover {
+  transform: scale(1.05);
+  transition: transform 0.5s ease-in-out;
+}
+
+/* div that contains landscape image with overcard */
+.landscape-overcard {
+  height: 30%;
+  width: 60%;
+  margin-left: 5%;
+}
+
+.overcard {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  align-self: flex-end;
+  padding: 0 var(--spacing-xl);
+  margin-bottom: 3.4rem;
+  position: absolute;
+  height: 35%;
+  width: 30%;
+  left: 58%;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  background-color: white;
+  box-shadow: var(--shadow-subtle);
+  border-radius: 1px;
+  font-family: "Gideon Roman", cursive;
+  color: var(--primary-color);
+}
+
+.opaque {
+  opacity: 0.9;
+}
+
+/* broader categories with overlay text */
+.category-overlay-card {
+  width: 33%;
+  aspect-ratio: 1/1;
+  overflow: hidden;
+  font-family: "Dancing Script", cursive;
+  font-size: 3rem;
+}
+
+/* div with overlay text */
+.text-overlay-card {
+  font-size: var(--fs-m);
+  position: absolute;
+}
+
+/* inner div for design implementation */
+.inner-category-overlay:hover {
+  transform: scale(1.05);
```

Comment:
you can group repeating properties together example .card, .note {transform: scale(0.9)};